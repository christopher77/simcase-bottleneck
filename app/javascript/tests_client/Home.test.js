import React from "react";
import { render } from "@testing-library/react";
import Home from "../views/home/Home";

test("Home view", () => {
  const { asFragment } = render(<Home />);
  expect(asFragment).toMatchSnapshot();
});
