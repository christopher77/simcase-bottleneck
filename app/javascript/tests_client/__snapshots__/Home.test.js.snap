// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home view 1`] = `[Function]`;
